Dockerfile

```diff
@@ -21,13 +22,17 @@ RUN \
                 && \
   rm -rf /var/lib/apt/lists/*
 
-# set java env
+# Java version
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 
-# Set Timezone to America/Los_Angeles
+# Perl dependencies
+RUN cpanm --installdeps .
```

Comment:
You are right!  Sorry for my mistake.
The README is confusing because it mentions files in the repository before giving instructions to clone the repository.  I think the README ought to be restructured to give instructions in the order they should be followed.  I have done that.

Reply:
(This is exactly what you suggested.)  Let me know if my edits look good.

Reply:
Yeah, it is looking good.